lubridate.Rmd

```diff
@@ -617,18 +617,32 @@ With indicator and renamed column
 ```{r sum NIEDERSCHLAGSDAUER - condition - Tag/Stunde on an interval of a day}
 df_sum2 <- df_group %>% 
   summarise(MENGE = sum(NIEDERSCHLAGSDAUER[NIEDERSCHLAGSINDIKATOR==1])) %>% 
+  filter(MENGE> 0)
   arrange( STATIONS_ID , TAG, STUNDE)
 head(df_sum2, 24) 
 ```
 
 Now we apply the knowledge with the indicator to the selection of autumn.
-Now we examine the duration of rainfall in autumn.
+First we examine the duration of rainfall in autumn.
 ```{r sum NIEDERSCHLAGSDAUER - condition - on an interval of season autumn}
-df_sum_autmn <- df_group_autmn %>% 
-  summarise(DAUER = sum(NIEDERSCHLAGSDAUER[NIEDERSCHLAGSINDIKATOR==1])) %>% 
-  arrange( STATIONS_ID , JAHR, MONAT, TAG)
+df_sum_autmn_group <- df_group_autmn 
+df_sum_autmn <- df_sum_autmn_group %>% 
+              summarise(DAUER = sum(NIEDERSCHLAGSDAUER[NIEDERSCHLAGSINDIKATOR==1])) %>% 
+              filter(DAUER>0) %>%
+              arrange( STATIONS_ID , JAHR, MONAT, TAG)
 head(df_sum_autmn, 10) 
 ```
+Now we examine the milliliter of rainfall.
```

Comment:
From the climate scientist perspective I would go with _precipitation_ rather than _rainfall_. The former also includes snow, hail, etc. 
In German it's like _Niederschlag_ vs _Regen_. 
But I might sound picky here as this is not the point of your chapter.